src/main/java/com/webonise/controllers/MainScreenController.java

```diff
@@ -0,0 +1,75 @@
+package com.webonise.controllers;
+
+import com.webonise.models.Coordinates;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.ProgressBar;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableView;
+import javafx.scene.control.cell.PropertyValueFactory;
+import javafx.scene.layout.Region;
+import javafx.scene.web.WebEngine;
+import javafx.scene.web.WebView;
+import netscape.javascript.JSObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.URL;
+import java.util.Observable;
+import java.util.Observer;
+import java.util.ResourceBundle;
+
+public class MainScreenController extends Region implements Initializable, Observer {
+
+    private static final Logger LOG = LoggerFactory.getLogger(MainScreenController.class);
+
+    private Coordinates coordinates;
+    private WebEngine webEngine;
+    private JSObject script;
+    private ObservableList<Coordinates> tableRows;
+
+    @FXML
+    private WebView mapBrowser;
+    @FXML
+    private ProgressBar progressBar;
+
```

Comment:
New line. Please be consistent. If you are adding new line for autowiring then here also u should add it.